chat_shell/chat_shell/tools/builtin/web_fetch.py

```
@@ -0,0 +1,257 @@
+    def _format_response(self, response: httpx.Response, url: str) -> str:
+        """Format HTTP response for LLM consumption."""
+        status_code = response.status_code
+        content_type = response.headers.get("content-type", "")
+        content_length = len(response.content)
+
+        if status_code >= 400:
+            error_body = ""
+            try:
+                error_body = response.text[:1000]
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)